include/sonar_driver/sonardevices/SonarImage.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <memory>
+#include <stdint.h>
+#include <vector>
+
+
+class SonarImage{
+public:
+    SonarImage() {
+        std::unique_ptr<std::vector<uint8_t>> data = std::make_unique<std::vector<uint8_t>>(1024);
+        std::unique_ptr<std::vector<int16_t>> bearingTable = std::make_unique<std::vector<int16_t>>(200000);
```

Comment:
Is this adequately sized? 400kB seems a bit large for a bearing table but maybe it's needed. I don't know.